OpenXmlFormats/OOXMLFactory.cs

```diff
@@ -22,7 +22,7 @@ public T Parse(Stream stream)
         }
         public T Create()
         {
-            return (T)Activator.CreateInstance(typeof(T));
+            return new T();
```

Comment:
Although `new T()` uses `Activator.CreateInstance` internally, this semantic is slightly different for AOT analyzers I believe.